notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<p>Need to spell out OSI SAF on first instance. See <a href="https://osi-saf.eumetsat.int/about-us" target="_blank">https://osi-saf.eumetsat.int/about-us</a>, should be Ocean Sea Ice Satellite Application Facilities?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Added to acronyms section</p>